### ISSUE TITLE
I don't use vagrant and i have this issue ?

### ISSUE THREAD

**Issue by softeg** [blinkreaction/drude#21]
I can't run dsh command's 

dsh st    

/home/asoo/Documents/projects/cn/.docker/bin/dsh: 30: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 192: [: ./: unexpected operator
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 128: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 30: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 202: [: unexpected operator
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 128: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 139: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 128: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 139: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 149: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
/home/asoo/Documents/projects/cn/.docker/bin/dsh: 155: /home/asoo/Documents/projects/cn/.docker/bin/dsh: [[: not found
"vagrant" executable not found.

**Comment by lmakarov** [comment]
@softeg what kind of shell do you have? `echo $SHELL`

**Comment by lmakarov** [comment]
BTW what is your OS?
`dsh` has not been much tested on Linux yet (outside of Tiny Core used by boot2docker).

**Comment by comxd** [comment]
Hi,

I have the same issue into Linux ((K)Ubuntu 14.10).

```
# echo $SHELL
/bin/bash
```

**Comment by comxd** [comment]
Can you test this commit on Windows and/or Mac?
https://github.com/blinkreaction/drude/compare/master...comxd:master

**Comment by lmakarov** [comment]
@comxd  can you change the script preamble to `#!/bin/bash` in `.docker/bin/dsh` and see if that works for you?
I have a workaround available now to fake `bash` in boot2docker, so there won't be a need to stick with `sh` standards and limitations.

**Comment by comxd** [comment]
Oh yes, it working good with "#!/bin/bash"!

**Comment by comxd** [comment]
But I have this error:

```
#docker-compose pull
Pulling db (blinkreaction/mysql:5.5)...
Couldn't connect to Docker daemon at http+unix://var/run/docker.sock - is it running?

If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
```

**Comment by lmakarov** [comment]
@comxd looks like your docker daemon  is not running.

Does `docker version` and `sudo docker run hello-world` work for you?

**Comment by comxd** [comment]
Sorry, I finally now can run this command without error :/

**Comment by frankcarey** [comment]
Doh, same issue here... wouldn't run properly in linux (ubuntu) until I made this change.

**Comment by frederickjh** [comment]
/bin/sh is a link to /bin/dash  in Ubuntu since 8.04 /bin/bash is still the user shell. My guess is these scripts are written on OSx which uses bash as sh.

**Comment by lmakarov** [comment]
Fixed in [v1.0.0-rc1](https://github.com/blinkreaction/drude/releases/tag/v1.0.0-rc1)
